### PR TITLE
Complete Tails 4 documentation and screenshots

### LINKED ISSUE #4893: Update docs for Tails 4
We want to 
- review all references in the documentation for accuracy against Tails 4/SecureDrop 1.1.0
- prioritize and update the highest priority issues (e.g., incorrect instructions or screenshots that are confusing) for 1.1.0.

This includes but is not limited to Tails references in the following sections:
- [Journalist Guide](https://docs.securedrop.org/en/release-1.0.0/journalist.html) - a lot of very specific Tails guidance here, e.g., regarding use of VeraCrypt
- [Printer setup docs](https://docs.securedrop.org/en/release-1.0.0/tails_printing_guide.html)
- [Install docs](https://docs.securedrop.org/en/release-1.0.0/install.html) ("Additional software" prompt)
- [SVS setup docs](https://docs.securedrop.org/en/release-1.0.0/set_up_svs.html)
- [Submission Key generation](https://docs.securedrop.org/en/release-1.0.0/generate_submission_key.html)
- [Network firewall setup](https://docs.securedrop.org/en/release-1.0.0/network_firewall.html) (e.g., "unsafe browser" use, network config on the AWS)

We should also sanity check:

- [Backup docs](https://docs.securedrop.org/en/release-1.0.0/backup_workstations.html)
- [Update docs](https://docs.securedrop.org/en/release-1.0.0/update_tails_usbs.html)
- [YubiKey docs](https://docs.securedrop.org/en/release-1.0.0/yubikey_setup.html)

### PR DESCRIPTION
**Pull request by rocodes**
Resolve #4893 
Add screenshots and check documentation for:
- Yubikey
- MAT2
- Veracrypt (Export device)
- Disks (Backup instructions)
- Misc updated UI (menu items etc) 

## Status

Ready for review

## Description of Changes

Docs only; screenshots and additional brief description of using MAT2 via CLI. Misc small changes to documentation as appropriate (eg, additional information on compatible Yubikey types).

Changes proposed in this pull request: Docs only

## Testing

Manual review

### If you made changes to documentation:

- [x] Doc linting (`make docs-lint`) passed locally

### CODE REVIEW

File: docs/journalist.rst [hunk 1]
```diff
@@ -438,27 +437,33 @@ highly sensitive submissions.
...
+
+To use MAT2 on the command line, type ``man mat2`` to see a list of available actions you can take with MAT2. For example, you can view the metadata of a file with ``mat2 myfile --show``.
+
+|mat2 cli show|
+
+You can create a "clean" version of the document with ``mat2 myfile``, again noting that this does not erase the metadata on the original file. This is equivalent to the "Remove metadata" context menu option.
+
+Note that even after running MAT2, you should carefully inspect files to ensure that all metadata has been wiped, or convert them to a simpler file format (for example, converting a ``.xls`` file to a ``.csv``) to ensure that metadata is not left behind in error.
+
+When you no longer need documents, in the Files application, you can right-click on them and choose **Wipe** to securely delete them.
```
Comment (zenmonkeykstop) [inline docs/journalist.rst]: That middle clause is a bit ambiguous, how about "When you no longer need documents, right-click  them in the Files application and choose **Wipe** to securely delete them."

File: docs/yubikey_setup.rst [hunk 2]
```diff
@@ -77,13 +79,15 @@ choose a location to save the log file. When the configuration is
...
+|YubiKey config successful|
+
 Adding Users
 ------------
 
 When adding new users, a SecureDrop admin will need the
 **Secret Key** value described above. She will enter it after
 selecting the **I'm Using a YubiKey** option while :ref:`adding users
-<Adding Users>`.
+<Adding Users>`. The new user will then have to verify their YubiKey before being added to the system. This means that the new user and the admin should be physically proximal for this process.
```
Comment (zenmonkeykstop) [inline docs/yubikey_setup.rst]: "present" vs "proximal"? Simpler language is better.

File: docs/admin.rst [hunk 3]
```diff
@@ -172,14 +172,13 @@ YubiKey
...
+the OATH-HOTP Secret Key that their YubiKey is configured with. For more
 information, read the :doc:`YubiKey Setup Guide <yubikey_setup>`.
 
 |Enable YubiKey|
 
-Once you've configured your YubiKey and entered the Secret Key, click
-*Add user*. On the next page, enter a code from your YubiKey by
-inserting it into the workstation and pressing the button.
+Once you've configured the YubiKey and entered the Secret Key, click
+*Add user*. On the next page, have the journalist enter a code from their YubiKey by inserting it into the workstation and pressing the button.
```
Comment (zenmonkeykstop) [inline docs/admin.rst]: Formatting to 80-char line width is preferred.

File: docs/backup_workstations.rst [hunk 4]
```diff
@@ -69,9 +69,11 @@ Fill out the form as follows:
 |Format Backup Drive|
 
 * **Erase**: `Don't overwrite existing data (Quick)`
-* **Type**: `Encrypted, compatible with Linux systems (LUKS + Ext4)`
+* **Type**: `Internal disk for use with other Linux systems only (Ext4)`, and make sure `Password protect volume (LUKS)` is checked
```
Comment (zenmonkeykstop) [inline docs/backup_workstations.rst]: Formatting to 80-char line width is preferred

File: docs/journalist.rst [hunk 5]
```diff
@@ -286,11 +286,10 @@ delete the files from your device:
 To decrypt and view documents or messages, return to your **Persistent** folder.
 All key actions are initiated by double-clicking:
 
-- Double-clicking archives in ZIP or gzip format will open the "File Roller"
-  application, which allows you to extract the contents.
+- Double-clicking archives in ZIP or gzip format will open the "Archive Manager"
+  application (called "file-roller" on the command line), which allows you to extract the contents.
```
Comment (zenmonkeykstop) [inline docs/journalist.rst]: Use ``file-roller`` instead? command line application names tend to be formatted as code.

File: docs/journalist.rst [hunk 6]
```diff
@@ -286,11 +286,10 @@ delete the files from your device:
...
 
-- Double-clicking files that end in ``.gpg`` will attempt to decrypt the contents
-  to the same directory. If you have configured a passphrase for your
+- Double-clicking files that end in ``.gpg`` will attempt to decrypt the contents to the same directory. If you have configured a passphrase for your
```
Comment (zenmonkeykstop) [inline docs/journalist.rst]: formatting to 80-char line width preferred.

File: docs/journalist.rst [hunk 7]
```diff
@@ -438,27 +437,33 @@ highly sensitive submissions.
 Removing Metadata
 ~~~~~~~~~~~~~~~~~
 
-.. tip:: For detailed information about removing metadata from documents, check out
-         this in-depth `guide to removing metadata`_.
+.. tip:: For detailed information about removing metadata from documents, check out this in-depth `guide to removing metadata`_.
```
Comment (zenmonkeykstop) [inline docs/journalist.rst]: as above, 80-char line width

File: docs/journalist.rst [hunk 8]
```diff
@@ -438,27 +437,33 @@ highly sensitive submissions.
...
 
-Tails comes with the `Metadata Anonymisation Toolkit`_ (MAT) that
+Tails comes with the `Metadata Anonymisation Toolkit 2`_ (MAT2) that
 is used to help strip metadata from a variety of types of files,
 including png, jpg, OpenOffice/LibreOffice documents, Microsoft Office
 documents, pdf, tar, tar.bz2, tar.gz, zip, mp3, mp2, mp1, mpa, ogg,
-and flac. You can open MAT by clicking **Applications** ▸ **System Tools** ▸
-**MAT**.
+and flac. We recommend using this and other tools to work with documents within Tails for as much of your workflow as possible.
```
Comment (zenmonkeykstop) [inline docs/journalist.rst]: 80-char thing applies here and below.

File: docs/yubikey_setup.rst [hunk 9]
```diff
@@ -13,10 +13,12 @@ What is a YubiKey?
 ------------------
 
 A YubiKey is a physical token used for two-factor authentication. They
-are made by a company called Yubico and are `commercially available`_.
+are made by a company called Yubico and are `commercially available`_. Note that not all physical tokens are compatible with the YubiKey Personalization Tool; for this, you require `a key that can support OATH-HOTP`_.
```
Comment (zenmonkeykstop) [inline docs/yubikey_setup.rst]: 80-char lines here too

Comment (rocodes) [reply docs/yubikey_setup.rst]: sorry about all the >80-line chars. I have it set to wrap at 80 but evidently not playing nice.

File: docs/journalist.rst (hunk 6 above)
Comment (rocodes) [reply docs/journalist.rst]: see 1c7e1817a

File: docs/yubikey_setup.rst (hunk 9 above)
Comment (rocodes) [reply docs/yubikey_setup.rst]: fixed in 1c7e1817a

File: docs/journalist.rst (hunk 5 above)
Comment (rocodes) [reply docs/journalist.rst]: fixed in 1c7e1817a

**Pull request merged**